contracts/contracts/harvest/Harvester.sol

```diff
@@ -391,17 +413,17 @@ contract Harvester is Governable {
         uint256 usdtBalance = usdt.balanceOf(address(this));
 
         uint256 vaultBps = 1e4 - tokenConfig.harvestRewardBps;
-        uint256 vaultShare = (usdtBalance * vaultBps) / 1e4;
+        uint256 rewardsProceedsShare = (usdtBalance * vaultBps) / 1e4;
 
         require(
             vaultBps > tokenConfig.harvestRewardBps,
-            "Address receiving harvest incentive is receiving more rewards than the vault"
+            "Address receiving harvest incentive is receiving more rewards than the rewards proceeds address"
         );
 
-        usdt.safeTransfer(vaultAddress, vaultShare);
+        usdt.safeTransfer(rewardProceedsAddress, rewardsProceedsShare);
```

Comment:
Not really related to this PR, but we should make it possible to pass in `address(0)` here for _rewardTo and skip the incentive logic and just transfer all the rewards to `rewardProceedsAddress`.

Reply:
Actually I mean if `_rewardTo == rewardProceedsAddress`